engine/src/main/java/org/terasology/entitySystem/prefab/internal/PojoPrefabManager.java

```diff
@@ -76,11 +77,7 @@ public boolean exists(String name) {
     public Collection<Prefab> listPrefabs(Class<? extends Component> comp) {
         Collection<Prefab> prefabs = Sets.newHashSet();
 
-        for (Prefab p : assetManager.getLoadedAssets(Prefab.class)) {
-            if (p.getComponent(comp) != null) {
-                prefabs.add(p);
-            }
-        }
+        prefabs.addAll(assetManager.getLoadedAssets(Prefab.class).stream().filter(p -> p.getComponent(comp) != null).collect(Collectors.toList()));
```

Comment:
This could probably be directly converted into a Set. Personally, I try to get lines shorter than 120 chars so the line is entirely visible in GitHub, but that's a matter of taste.